{
	"id": "CVE-2016-9013",
	"vulnerabilities": [
		{
			"id": "CVE-2016-9013",
			"references": [
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.alpinelinux.org/vuln/CVE-2016-9013"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "alpine:edge",
			"criteria": {
				"operator": "OR",
				"criterions": [
					{
						"vulnerable": true,
						"package": {
							"name": "py3-django",
							"architectures": [
								"aarch64",
								"armhf",
								"armv7",
								"ppc64le",
								"riscv64",
								"s390x",
								"x86",
								"x86_64"
							],
							"repositories": [
								"community"
							]
						},
						"affected": {
							"type": "apk",
							"range": [
								{
									"lt": "1.8.16-r0"
								}
							],
							"fixed": [
								"1.8.16-r0"
							]
						}
					}
				]
			}
		}
	],
	"data_source": "alpine-secdb"
}
